src/components/spreadsheet/utils/cell-renderers.js

```diff
@@ -64,7 +64,7 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 export const BooleanCellRenderer = (props) => {
-    const isChecked = props.value;
+    const isChecked = Boolean(props.value);
```

Comment:
Not linked to the initial issue but fixes a warning when updating a boolean value